docs/user-guide/spin.rst

```diff
@@ -7,7 +7,10 @@ Original paper: https://arxiv.org/abs/2401.01335
 
 The NeMo framework supports efficient model alignment via the NeMo Aligner codebase.
```

Comment:
fix capitalization and punctuation.

The NeMo Framework supports efficient model alignment via the NeMo-Aligner codebase.